test/e2e/tls/tls_test.go

```diff
@@ -18,14 +18,24 @@ import (
 var testRuntime *e2e.TestRuntime
 var pool *x509.CertPool
 
+var minTLSVersions = map[string]uint16{
+	"1.0": tls.VersionTLS10,
+	"1.1": tls.VersionTLS11,
+	"1.2": tls.VersionTLS12,
+	"1.3": tls.VersionTLS13,
+}
+
 // print error to stderr, exit 1
 func fatal(err interface{}) {
 	fmt.Fprintf(os.Stderr, "%s\n", err)
 	os.Exit(1)
 }
 
 func TestMain(m *testing.M) {
+	minTLSVersion := flag.String("--min-TLS-Version", "1.2", "minimum TLS Version")
+	TLSVersion := minTLSVersions[*minTLSVersion]
 	flag.Parse()
+	var err error
```

Comment:
Why do we need this variable? 🤔

Reply:
yeah, doesnt make sense to have it, I will remove it